CHANGES.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+# 1.1.6 (core) / 0.17.6 (libraries)
+
+### New
+
+- [dagster-dbt] When attempting to generate software-defined assets from a dbt Cloud job, an error is now raised if none are created.
+- [dagster-dbt] Software-defined assets can now be generated for dbt Cloud jobs that execute multiple commands.
+- Throughout Dagit, when the default repository name `__repository__` is used for a repo, only the code location name will be shown. This change also applies to URL paths.
+
+### Bugfixes
```

Comment:
could we also order this section so we have the core entries first and integrations later